### PR TITLE
Thankyou page added need review

### PR DESCRIPTION
**Pull request by Amutha37**
Thank you page added to the pages . I have not tested the layout. Question 5 path on submit taking it to result page. We need to change that to Thankyou page.

### CODE REVIEW

File: pages/form/Thankyoupage.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { Heading, Text, Box } from "@chakra-ui/react";
+import Layout from "../../components/Layout/Layout";
+
+export default function Question5() {
```
Comment (batteries76) [inline pages/form/Thankyoupage.jsx:4]: Could you rename this function please?

**Review by batteries76 — CHANGES_REQUESTED** [review]
Great work Amutha!
1. Please rename the function.
2. Could you please camel-case the file name in the file system.
3. The cooler of the text should be white on a blue background in the Figma? Am I right about that?
4. Are you able to have another go at getting the cloud in? We might be able to run through it later.

File: pages/form/Thankyoupage.jsx (hunk 1 above)
Comment (Amutha37) [reply pages/form/Thankyoupage.jsx:4]: OOOPS... How did my eye miss that. Fixed. Pushing the changes now.

**Comment by hqtan** [comment]
[DEV [MVP] New designs in thank you page](https://trello.com/c/WWyHAIaG/214-dev-mvp-new-designs-in-thank-you-page)

**Pull request closed without merging**